client/app/utils/i18n.js

```diff
@@ -0,0 +1,7 @@
+function translate(translations) {
```

Comment:
IMO being inside the `utils` directory is clear enough. Any single word could be a popular JavaScript project, and those would be located in some other  directory anyways.

However, I would have a convention to import any util file such as `utils/foo.js` as `fooUtils` if the whole module is imported to a "namespace" object like `import * as fooUtils from 'util/foo'`.